### PR TITLE
fix double click bug for swiping

### PR DESCRIPTION
**Pull request by mksavic**
### Description

Only 1 click is required to update the stack at `/<eid>/match`.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1.  Mark some users as interested in / going to the same event.
1.  Navigate to `/<eid>/match`. Click "Thumbs Up" or "Thumbs Down". See that the user disappears from the stack after the first click. Verify in `/admin` that the DB updates accordingly. Once #145 is accepted, a popup will appear on matches as well (I tested this but decided to keep the PRs separate).

### Miscellaneous Comments

*   Basically the issue was that the `users` variable was not being updated during a POST.

### Checklist

- [x] I ran `git merge develop` before submitting this PR.
- [x] I understand that I may have to run `git merge develop` again after submitting this PR since new changes may have been pulled into `develop`.
- [x] I understand that it is MY responsibility to make sure that this PR does not have any conflicts.

### CODE REVIEW

File: showup/views.py [hunk 1]
```diff
@@ -67,37 +67,38 @@ def edit_profile(request, id):
...
+def get_stack(request, eid):
+    # My uid.
+    uid = request.user.id
+
+    # Get the users interested in or going to the event.
+    users = (CustomUser.objects.filter(interested__id=eid) |
+             CustomUser.objects.filter(going__id=eid))
+
+    # These are the users that swiped left on me.
+    swiped_left = [x.swiper.id for x in Swipe.objects.filter(
```
Comment (kavinshah) [inline showup/views.py]: please don't change variable names. unless you're removing the code entirely. This affects all current branches and PRs

**Comment by kavinshah** [comment]
functionality works otherwise.

File: showup/views.py (hunk 1 above)
Comment (mksavic) [reply showup/views.py]: Not sure what you mean. I pulled this code and made an entirely different function (`get_stack()`) so changing the variable names won't cause any errors.

Comment (kavinshah) [reply showup/views.py]: my bad. git highlighted looked like that variable have been renamed.
